source/Core/BSP/BSP.h

```diff
@@ -92,7 +92,7 @@ enum StatusLED {
   LED_COOLING_STILL_HOT, // The unit is off and cooling but still hot
   LED_UNKNOWN,           //
 };
-void setStatusLED(const enum StatusLED state);
+void setStatusLED(const enum StatusLED state, bool buzzer);

```

Comment:
The Buzzer controls were already part of the LED code though? That's why i kept it that way.

I can move it completely out of there of course.

Reply:
There is an existing setBuzzer function designed around the buzzer.
Mostly want them seperate so that devices that only have one or the other are a bit less confusing.

Reply:
Done